dotcom-rendering/README.md

```diff
@@ -39,7 +27,6 @@ $ make dev
 ```
 
 `make dev` will start the development server on port 3030: [http://localhost:3030](http://localhost:3030).
-`make build && make start` will start the production server on port 9000: [http://localhost:9000](http://localhost:9000).
```

Comment:
We very rarely need to do this

Reply:
There is no `start` in the makefile so this can definitely be removed!